packages/ui/ProgressBar/ProgressBar.module.less

```diff
@@ -35,6 +42,11 @@
 			bottom: ~"calc(var(--ui-progressbar-proportion-start-background) * 100%)";
 			height: ~"calc(var(--ui-progressbar-proportion-end-background) * 100%)";
 		}
+
+		.separator {
+			width: 48px;
```

Comment:
My initial thought is that there shouldn't be any style applied in `ui`.  If we really felt there needed to be, these should become LESS variables and placed into the appropriate less file.

Reply:
The values are move from `ui` to `sandstone`